Status Update 2020-07-10 (#89)
puppetlabs/iac: _posts/status_update_2020_07_10.md

---
layout: post
title: "10th July 2020: IAC Team Status Update"
author: adrianiurca
categories:
  - team
  - status
tags:
  - releases
  - community_management
  - iis
  - tomcat
  - powershell
  - shoutouts
---

This has been as short week for the IAC team and we mostly focused our attention on maintenance and support activities.
We want to share with our community some of the most interesting things.
Here’s a brief overview of what we’ve been up to:

## Module Releases from IAC Team

- [puppetlabs/iis](https://github.com/puppetlabs/puppetlabs-iis)(7.1.0)
- [puppetlabs/tomcat](https://github.com/puppetlabs/puppetlabs-tomcat)(4.1.0)
- [puppetlabs/powershell](https://github.com/puppetlabs/puppetlabs-powershell)(4.0.0)

> ## Community Shout outs for IAC

Reviewer: ```suggestion
## Community Shoutouts for IAC
```

We appreciate our community contributors as they are always ready to help and they have excellent ideas. Here are some highlights of this week:

Thanks to [Ben Ford](https://github.com/binford2k) for his great idea [feature](https://github.com/puppetlabs/community_management/pull/39) on community_management.

Thanks to [Chris Denneen](https://github.com/cdenneen) for the contributions on pdk-templates([#338](https://github.com/puppetlabs/pdk-templates/pull/338), [#337](https://github.com/puppetlabs/pdk-templates/pull/337), [#332](https://github.com/puppetlabs/pdk-templates/pull/332))

> ## Customer tickets this week

Reviewer: ```suggestion
## Customer Tickets this Week
```

The team works hard at Docker issue ([MODULES-10691](https://tickets.puppetlabs.com/browse/MODULES-10691)).
We have a PR opened for this issue [PR](https://github.com/puppetlabs/puppetlabs-docker/pull/632).
Many thanks to [Daiana](https://github.com/daianamezdrea) for managing this problem.

> ## Michael and Sheena are getting back to DCS revamp work after a hard triage

Reviewer: ```suggestion
## Michael and Sheena are getting back to DSC revamp work after a hard triage
```

layout: post
title: "10th July 2020: IAC Team Status Update"
author: adrianiurca
categories:

- team
- status

>   tags:

Reviewer 2: ```suggestion
tags:
```

- releases
- community_management
- iis
- tomcat
- powershell
- shoutouts

---

> The IAC team has been working on a lot of things over the past week.

Reviewer 3: ```suggestion
This has been as short week for the IAC team and we mostly focused our attention on maintenance and support activities.
```

We want to share with our community some of the most interesting things.
Here’s a brief overview of what we’ve been up to:

## Module Releases from IAC Team

- [puppetlabs/iis](https://github.com/puppetlabs/puppetlabs-iis)(7.1.0)

> - [puppetlabs/java_ks](https://github.com/puppetlabs/puppetlabs-tomcat)(4.1.0)

Reviewer 4: ```suggestion
- [puppetlabs/tomcat](https://github.com/puppetlabs/puppetlabs-tomcat)(4.1.0)
```

- [puppetlabs/powershell](https://github.com/puppetlabs/puppetlabs-powershell)(4.0.0)

## Community Shout outs for IAC

We appreciate our community contributors as they are always ready to help and they have excellent ideas. Here are some highlights of this week:

Thanks to [Ben Ford](https://github.com/binford2k) for his great idea [feature](https://github.com/puppetlabs/community_management/pull/39) on community_management.

> Thanks to [Chris Denneen](https://github.com/cdenneen) for the contributions on pdk-templates.

Reviewer 2: I would perhaps link to the PRs from Chris too

## Customer tickets this week

The team works hard at Docker issue ([MODULES-10691](https://tickets.puppetlabs.com/browse/MODULES-10691)).
We have a PR opened for this issue [PR](https://github.com/puppetlabs/puppetlabs-docker/pull/632).
Many thanks to [Daiana](https://github.com/daianamezdrea) for managing this problem.

## Michael and Sheena are getting back to DCS revamp work after a hard triage

> @TODO

Reviewer 4: @michaeltlombardi Could you please confirm this . 
Will be working on a handcrafted automated end-to-end test which validates autogenerated modules are ready for production.

## Happy 90 days at Puppet

A great achievement made by [Adrian](https://github.com/adrianiurca).

> For his hard work made in these 3 months, he was rewarded to write this blog post.

Reviewer 3: ```suggestion
For the great progress he made in this time, as a reward, he got to write this week's status update.
```
